pkg/storage/s3/getter.go

```diff
@@ -100,6 +100,9 @@ func (s *Storage) open(ctx context.Context, path string) (io.ReadCloser, error)
 	if err != nil {
 		return nil, errors.E(op, err)
 	}
-
-	return goo.Body, nil
+	var size int64
+	if goo.ContentLength != nil {
```

Comment:
I'm assuming that `ContentLength` is never expected to be nil, but if it is, is it ok that we return a 0 length to the doc server? looks like it will be more likely to [process the module](https://github.com/golang/pkgsite/blob/fd89b15b08609658e2b6d1363f52fd2f691db311/internal/fetch/fetch.go#L173) even if it's big

Reply:
@arschles yep, I think we should be fine. I contemplated putting an `if zip.Size() > 0` on the handler level so we only write the Content-Length header when size is greater than zero, but I figured if S3 is not returning content-length there are much more important things to worry about (since S3 is down or buggy) than the content-length header?